types/database/draft.d.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Model, {Associations} from '@nozbe/watermelondb/Model';
+
+/**
+ * The Draft model represents  the draft state of messages in Direct/Group messages and in channels
+ */
+export default class Draft extends Model {
+    /** table (entity name) : Draft */
+    static table: string;
+
+    /** associations : Describes every relationship to this entity. */
+    static associations: Associations;
+
+    /** channel_id : The foreign key pointing to the channel in which the draft was made */
+    channelId: string;
+
+    /** message : The draft message */
+    message: string;
+
+    /** root_id : The root_id will be empty for Direct Message but have a value for draft replies of a thread */
```

Comment:
Yes.  The ` root_id ` will hold a value only for threaded conversations; else it will be empty.  I will update this comment and the one in the draft model as well.